aclk/aclk_lws_wss_client.c

```diff
@@ -0,0 +1,340 @@
+#include "aclk_lws_wss_client.h"
+
+#include "libnetdata/libnetdata.h"
+
+#define UNUSED(x) (void)(x)
+
+static int aclk_lws_wss_callback(struct lws *wsi, enum lws_callback_reasons reason, void *user, void *in, size_t len);
+
+struct aclk_lws_wss_perconnect_data {
+    int todo;
+};
+
+//TODO config??
+int aclk_lws_wss_use_ssl = 1;
```

Comment:
I don't think we have a use-case that does not use ssl.

Reply:
i removed it in PR #7988 as that is where it belongs